### DISCUSSION TITLE
M1 Mac Homebrew FreeCAD installs but MacOS says that...

### DISCUSSION THREAD

**Discussion by superurbi** [Homebrew/discussions#936] [Getting started]
Hi All,
Correctly installed but MacOS says that it can't verify if the developer so it invites me to drop it to the trash bin.
regards,

**Comment by mahdyar** [comment]
After the dialog box shows up:
1. Click `ok`.
2. Open `System Preferences`.
3. Go to `Security and Privacy -> General`.
4. There will be a message about the program that was blocked, with an option to `Open Anyway`.

According to [here](https://support.apple.com/en-us/HT202491), it should solve the problem.
